networks/mainnet.json

```diff
@@ -61,7 +64,10 @@
         "Pair_BAT_ETH": "0xb6909b960dbbe7392d405429eb2b3649752b4838",
         "Pair_ETH_USDC": "0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc",
         "PriceFeedPoster": "0x83563ba7f1b093aae57fe876f8d870f8a1508886",
-        "CrowdProposalFactory": "0xB5212a2fa63c1863b9e8670e2A6D420d0309c502"
+        "CrowdProposalFactory": "0xB5212a2fa63c1863b9e8670e2A6D420d0309c502",
+        "StdComptrollerG5": "0x7b5e3521a049C8fF88e6349f33044c6Cc33c113c",
```

Comment:
this in here 2x?

Reply:
No those are remove/add fixing the missing comma

Reply:
StdComptrollerG5 is on line 45 & 67 both

Reply:
🎣